packages/hub/src/app/ai/Sidebar.tsx

```diff
@@ -147,15 +170,54 @@ Outputs:
             </StyledTab.Panels>
           </div>
         </StyledTab.Group>
-        <SelectStringFormField<FormShape, LlmId>
-          name="model"
-          label="Model"
-          size="small"
-          options={MODEL_CONFIGS.filter(
-            (model) => model.provider === "anthropic"
-          ).map((model) => model.id)}
-          required
-        />
+        <div className="flex flex-col gap-2">
+          <div className="flex items-center justify-between">
+            <TextTooltip
+              text="Enhance the model's numerical calculations and reasoning."
+              placement="bottom"
+            >
+              <span className="cursor-help text-sm underline decoration-dotted">
+                Numeric Steps
+              </span>
+            </TextTooltip>
+            <span className="w-16">
+              <NumberFormField<FormShape> name="numericSteps" size="small" />
+            </span>
+          </div>
+          <div className="flex items-center justify-between">
+            <TextTooltip
+              text="Improve the model's documentation and formatting."
+              placement="bottom"
+            >
+              <span className="cursor-help text-sm underline decoration-dotted">
+                Documentation Steps
+              </span>
+            </TextTooltip>
+            <span className="w-16">
+              <NumberFormField<FormShape> name="styleGuideSteps" size="small" />
+            </span>
+          </div>
+        </div>
+        <div className="flex items-center justify-between">
+          <TextTooltip
+            text="Choose the LLM to use. Sonnet is much better than Haiku, but is around 3x more expensive. We use the latest versions of Sonnet 3.5 and Haiku 3.5."
+            placement="bottom"
+          >
+            <span className="cursor-help text-sm underline decoration-dotted">
+              Model
+            </span>
```

Comment:
Also tbh I'm not entirely comfortable with inventing a separate type of form field labels for this, instead of reusing the common code from `@quri/ui`:

- proper form field labels have the advantage of "when you click on the label, the field will be focused"
  - plus some vague accessibility benefits
  - and consistent styling guarantees
- `@quri/ui` form fields already support labels and have an [`inlineLabel` flag](https://quri-ui.vercel.app/?path=/docs/ui-forms--docs#generic-field-components) for left-to-right layout, but they don't support tooltips
- so I propose we add the support for `labelTooltip` field there, it shouldn't be very hard, mostly passing props around

Reply:
> it shouldn't be very hard, mostly passing props around

I know types in `@quri/ui` might seem gnarly, but it's really not that bad, I checked out the code to make sure it's doable.

You basically just need to trace where `inlineLabel` or `description` are passed around and add another field for those, then support the new field in `FormFieldLayout.tsx`.

For `SelectFormField` and `SelectStringFormField` you'll also need to add it to the `Pick<FormFieldLayoutProps, "label" | "description">`, along with `inlineLabel` which was unsupported by those.

Reply:
I didn't see the `inlineLabel` flag. I'll give this a quick go. At the same time, the css here is very simple / easy to do, and I don't care too much about standardization now (I lean towards velocity for a lot of this stuff).

Reply:
Doing a bit of digging... this one seems more confusing. Might address later.